### ISSUE TITLE
Problem with some feeds

### ISSUE THREAD

**Issue by Greg-1981** [FreshRSS/FreshRSS#6660]
Hi, 

Switching from QuiteRSS, I juste insalled FreshRSS, but some of my feeds are not functionning.

Examples of that feeds :

https://www.telos-eu.com/fr/rss.xml
https://www.olivierdauvers.fr/feed/
https://www.actu-environnement.com/ae/news/archives/rss.php4

All are OK according to W3C ; I had no problem with QuiteRSS, and I also tried on the FreshRSS online demo, where it runs fine too.

So, something is wrong with my configuration, but what ?

I am using FreshRSS 1.24.1 on shared host with PHP 7.4 and MySQL 8.0

All is green according to FreshRSS.

In my logs, concenring the feeds I mentionned before :

cURL error 28: Connection timed out after 20000 milliseconds

Unknown error for feed 

Thanks for your help, and please, apologize for my bad english.

**Comment by Frenzie** [comment]
How long has it been? It could be as simple as the connection to those servers being interrupted for some reason.

**Comment by Greg-1981** [comment]
> How long has it been? It could be as simple as the connection to those servers being interrupted for some reason.

Maybe for five hours. As soon as I imported my feed list in fact. But if their servers were itnerrupted, I guess I wouldn't be able to connect to them with Firefox (RSSpreview plugin), or with the FeshRSS online demo either.

**Comment by Greg-1981** [comment]
For the first example feed, well, things just started working after clicking on “reload”.

For the other two, I had the idea of testing a simple PHP script supposed to display RSS feeds, and that didn't work either. Perhaps my host's IP could be blacklisted ? I didn't think of that right away.

**Comment by Alkarex** [comment]
All feeds work for me.
As a first test, try to curl from your server `curl -v 'https://www.olivierdauvers.fr/feed/'` and with https://validator.w3.org/feed/
See https://github.com/FreshRSS/FreshRSS/discussions/6287

**Comment by Greg-1981** [comment]
I have to say that given the use of shared hosting and my modest skills, I would not know how to execute the command. I could find out how to do it (I haven't looked yet), but at first, I tried to access the feeds by very simple PHP scripts, which confirmed my idea that I was blocked by certain sites without FreshhRSS being the cause.

However, now that I had this in mind, I could understand that other users sometimes had the same problem with certain feeds and I saw that some people got by by creating an intermediate feed in Feedburner. This is not a completely satisfactory solution, but in practice, for the moment, it seems to work, and it allows me to test FrshRSS without sacrificing any information feed.

Note: one or two of my feeds (probably badly formatted) were not accepted by Feedburner either but were accepted in RSS App. I am reporting this for the benefit of those who were experiencing the same kind of difficulties.
